noisepagecontrol/control_plane/services/exploratory_worker_handler/collect_data_from_exploratory.py

```diff
@@ -0,0 +1,42 @@
+import json
+import logging
+import requests
+
+from django.conf import settings
+
+from control_plane.services.resource_manager.initialise_resource import (
+    initialise_resource,
+)
+from control_plane.services.resource_manager.resource_type import ResourceType
+
+logger = logging.getLogger("control_plane")
+
+
+def collect_data_from_exploratory(
+    tuning_id,
+    event_name,
+    replica_url,
+    postgres_port,
+    data_collector_type,
+    data_collector_config,
+):
+    logger.info(f"Sending request to collect data. Event name: {event_name}")
+
+    # Initialise a new resource for this workload
+    resource_id = initialise_resource(tuning_id, ResourceType.CATALOG)
```

Comment:
ResourceType should depends on the collector_type parameter?

Reply:
Good point, it might be difficult to differentiate it here; should we rename it to `ResourceType.EXPLORATORY_DATA` ?